### ISSUE TITLE
SimpleRecord::Attributes#get_attribute is very slow

### ISSUE THREAD

**Issue by krishan1390** [appoxy/simple_record#56]
The performance of the get_attribute method for simple record is pretty bad. Profiling results show the code spends 1.65 secs in this method for 8050 calls. Surely something is wrong

**Comment by treeder** [comment]
Are you using clob types?

**Comment by krishan1390** [comment]
No. All the data types used are simple integer and string data types. And every field fetch takes more time than expected
